source/atlas-cli-changelog.txt

```diff
@@ -14,6 +14,22 @@
    :depth: 1
    :class: singlecol
 
+.. _atlas-cli-1.33.0:
+
+{+atlas-cli+} 1.33.0
+--------------------
+
+Released 4 December 2024
+
+- Adds the ``--mdbVersion`` and ``--connectWith`` flags in the :ref:`atlas-setup`
+  command.
+- Deprecates the ``--skipMongosh`` flag in the :ref:`atlas-setup` command.
+  When you use the :ref:`atlas-setup` and :ref:`atlas-deployments-setup`
+  commands in interactive mode, the commands now behave in the same way
+  when you choose ``default`` (Deploy with default settings),
+  ``custom`` (Deploy with custom settings), or ``cancel`` (Cancel setup)
+  options.
```

Comment:
I'm confused by this statement... what do we mean by "behave in the same way"? Does it mean that all of those choices do the same thing? Or does it mean that all of those choices can't skip mongosh? I'm assuming it's the latter... can we confirm and make it more clear?

Also, minor nit:
```suggestion
  commands in interactive mode, the commands now behave in the same way
  when you choose ``default`` (deploy with default settings),
  ``custom`` (deploy with custom settings), or ``cancel`` (cancel setup)
  options.
```

Reply:
Yes, I too was confused, and thought you'd know. I will check with devs.

Reply:
This is the PR https://github.com/mongodb/mongodb-atlas-cli/pull/3410 that adds this change. Here we meant that now the interactive modes of `atlas setup` and `atlas deployment setup` are the same (ask the same questions, use the same default values, behave in the same way).